### ISSUE TITLE
As a somewhat frequent user, I wish for opening the same file/storage repeatedly to be easier

### ISSUE THREAD

**Issue by SacredDuckwhale** [WowDevTools/WDBXEditor#21]
I usually do a "Load from CASC" for the same source (the WOW client). However, every time I do that I have to fully navigate the entire file system in my search for the folder, regardless of whether or not it has changed. This is taking a lot of time and it's unnecessarily annoying, making working with the editor on a regular basis a bit of a hassle.

A few suggestions to resolve this issue:
* Add a "Recent files" feature, as many programs do, allowing me to open the same few files/containers more easily. It's not a problem if it has to rescan, so obviously I wouldn't expect it to save anything more than the path/file names; it's the selecting that takes most of the time anyway
* Change the "Open from CASC" (and, presumably, similar dialogs) to use a different selection system, so that paths can be copied and pasted at the very least. There's an edit box right next to the "Browse" button, but alas - it is read-only!

Unfortunately I am clueless when it comes to .NET, which means I am unable to provide a solution that is ready to merge via PR at this time :(

*Edit: I just noticed that if the opening process fails, the path is even saved in the edit box. However, there is no way to re-open it and try again, so all that remains is to click "Browse" and navigate the entire path once more. Maybe adding a second button to open the path given in said box would be helpful? If the box was made writeable then it could also be used to manually enter the path instead of having to select it via the terrible GUI element.*

PS: If there's an easier way that I missed, please enlighten me!

**Comment by barncastle** [comment]
I've added the following:

1. Persistent saving of the last used CASC/MPQ path used from the "Load from ..." screen. This populates on screen load.
2. A 'Recent Files' list to the 'File' menulist for any files that are opened directly. This is capped to the last 10 most recent.
3. A 'Load' button on the "Load from ..." screen to reload the currently selected archive and to load point 1.

If there is anything else let me know. I'll release this after I've closed the other issue.
